students/etwum/lesson08/circle_class.py

```diff
@@ -0,0 +1,52 @@
+from math import pi
+
+
+class Circle():
+
+    def __init__(self,radius):
+        self.radius = radius
+        self.__area = pi * radius**2
+
+    @property
+    def diameter(self):
+        return self.radius * 2
+
+    @diameter.setter
+    def diameter(self, radius):
+        self.radius = radius/2
+
+    @property
+    def area(self):
+        return self.__area
```

Comment:
As noted above, I would move the operation here:
```
    @property
    def area(self):
        return pi * self.radius**2
```
And because properties defined in this way are read-only, there's no need for the leading double underscore on the variable name. `circle.area = 6` will throw an `AttributeError` if you don't define a setter for area